Source/options.cpp

```diff
@@ -701,7 +701,7 @@ std::vector<OptionEntryBase *> ChatOptions::GetEntries()
 }
 
 OptionEntryLanguageCode::OptionEntryLanguageCode()
-    : OptionEntryListBase("Code", OptionEntryFlags::CantChangeInGame | OptionEntryFlags::RecreateUI, "Language", "Define what language to use in game")
+    : OptionEntryListBase("Code", OptionEntryFlags::CantChangeInGame | OptionEntryFlags::RecreateUI, "Language", N_("Define what language to use in game."))

```

Comment:
Let's do this for `"Language"` as well?

Reply:
"Language" was already there, so we did not apply it.

Reply:
If the other line is removed it would cause the string to no longer be translated, so best to have it marked explicitly.

```suggestion
    : OptionEntryListBase("Code", OptionEntryFlags::CantChangeInGame | OptionEntryFlags::RecreateUI, N_("Language"), N_("Define what language to use in game."))
```